contrib/completions/bash/buildah

```diff
@@ -357,20 +357,11 @@ return 1
      --build-arg
      -c
      --cert-dir
```

Comment:
are we removing these from buildah bud?  Don't we need to add them to buildah run?